examples/quickstart.php

```diff
@@ -34,14 +34,11 @@ class User extends AggregateRoot
 
         /**
          * ARs should be created via static factory methods
-         *
-         * @param string $username
-         * @return User
          */
-        public static function nameNew($username)
+        public static function nameNew(string $username): User
         {
             //Perform assertions before raising a event
-            \Assert\that($username)->notEmpty()->string();
+            Assertion::notEmpty($username);
```

Comment:
I did not want to repeat all requirements from prooph/common in all components, since it's already a dependency there. This also helps updating dependencies, as we only have to do it at one place.